ocs_ci/utility/deployment.py

```diff
@@ -228,3 +230,29 @@ def workaround_mark_disks_as_ssd():
             logger.info("Workaround already applied.")
         else:
             raise err
+
+
+def create_openshift_install_log_file(cluster_path, console_url):
+    """
+    Workaround.
+    Create .openshift_install.log file containing URL to OpenShift console.
+    It is used by our CI jobs to show the console URL in build description.
+
+    Args:
+        cluster_path (str): The path to the cluster directory.
+        console_url (str): The address of the OpenShift cluster management-console
+    """
+    installer_log_file = os.path.join(cluster_path, ".openshift_install.log")
+    formatted_time = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+
+    logger.info(f"Cluster URL: {console_url}")
+    with open(installer_log_file, "a") as fd:
+        fd.writelines(
+            [
+                "W/A for our CI to get URL to the cluster in jenkins job. "
+                "Cluster is deployed via Assisted Installer API!\n"
```

Comment:
I did not change this line, I only pushed create_openshift_install_log_file function to common space from [ocs_ci/deployment/assisted_installer.py](https://github.com/red-hat-storage/ocs-ci/pull/10703/files#diff-9948d23846f98e69566cb5c772b5caeedf1b888adacaec090f0616cd92c80f10) 
as far as I understand it is a mock-up for our log-parser on Jenkins pipelines.

TBH, correcting code(mocked-up log message) for our custom-built pipelines that are not validating this log message other than retrieving the console should have a very low priority. Until it can be important for something else.

Reply:
For the Jenkins job, the important line is only the last one `... Access the OpenShift web-console here ...`.

The think is, that initially it was used only for the Assisted Installer deployment, where it make sense the message, but since you are moving it to the common space and re-using it for the ROSA deployment also, it will be worth to update the message to avoid confusion.
I think it is not necessary to make it dynamic based on the actual deployment type, but just more generic. What about something like this?

```
"Cluster is deployed via some kind of managed deployment (Assisted Installer API or ROSA). OpenShift Installer (IPI or UPI deployment) were not used!\n"
```